src/components/application_manager/src/resumption/resume_ctrl_impl.cc

```diff
@@ -336,6 +336,10 @@ void ResumeCtrlImpl::OnSuspend() {
 void ResumeCtrlImpl::OnIgnitionOff() {
   LOG4CXX_AUTO_TRACE(logger_);
   if (!application_manager_.IsLowVoltage()) {
+    if (!resumption_storage_) {
```

Comment:
@mkorniichuk do we have case if resumption storage may be nit initialized? If no> I would put DCHECK here

Reply:
@mkorniichuk in case if FS is read only, it should be handled inside resumption storage, isn't it? Or even in Last state.

Reply:
@LuxoftAKutsan DCHECK will stop application immediately with logged fatal error. But we still need to stop all components, don't we?
About the FS. It is handled in application manager Init(). So the Init() of resumption_storage is not even called. Then StopComponents takes place, where Stop method is called for application manager, and so the OnIgnitonOff method of ResumptionController is called as well. And since resumption_storage is not initialized, a crash will occur.

Reply:
DCHECK is assertion. Assertion will be removed from release build. During testing if we found such condition, we can just crash SDL with appropriate information.